CHANGELOG.rst

```diff
@@ -4,6 +4,12 @@ Changelog
 in development
 --------------
 
+3.4.0 
+-----
```

Comment:
Please remove the 3.4.0. 

```suggestion
Added
~~~~~
```
The Changelog record comes under the "development" section and is relevant to "Added" category. Everything else related to version cutting will be handled during the release preparation by maintainers.